js/errors.js

```diff
@@ -1,4 +1,5 @@
 
+    module.ErmrestError = ErmrestError;
```

Comment:
This is not really important. But the module that we are exposing is `ERMrset`, so just replace all these `ErmrestError` with `ERMrestError.`